packages/theme/pages/Product.vue

```diff
@@ -372,21 +355,21 @@ export default {
     );
     display: flex;
     align-items: center;
-    margin-top: var(--spacer-xl);
+    margin-top: var(--spacer-sm);
   }
   &__color-label {
-    margin: 0 var(--spacer-lg) 0 0;
+    margin: 0 var(--spacer-sm) 0 0;
   }
   &__color {
     margin: 0 var(--spacer-2xs);
   }
   &__add-to-cart {
     margin: var(--spacer-base) var(--spacer-sm) 0;
```

Comment:
Could we change the __add-to-cart margin from `margin: var(--spacer-base) var(--spacer-sm) 0;` to `margin: var(--spacer-base) 0 var(--spacer-sm);`? This horizontal margin applied on buttons looks awkward and breaks the layout. Aditionally, could we increase the negative space to reduce the overwhelming visual squeeze between the buttons and "select-size" section? I would suggest applying `--spacer-2xl`, so that we eventually end up with:
```
&__add-to-cart {
  margin: var(--spacer-2xl) 0 var(--spacer-sm);
}
 ```

Reply:
Good idea, I applied it